flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/converters/AbstractCreateMaterializedTableConverter.java

```diff
@@ -184,7 +185,7 @@ private void verifyPartitioningColumnsExist(
             if (schema.getColumn(partitionKey).isEmpty()) {
                 throw new ValidationException(
                         String.format(
-                                "Partition column '%s' not defined in the query schema. Available columns: [%s].",
+                                "Partition column `%s` not defined in the query's schema. Available columns: [%s].",
```

Comment:
nit: use single quotes for consistency
```suggestion
                                "Partition column '%s' not defined in the query's schema. Available columns: [%s].",
```